python-lib/dku_plugin_config_loading.py

```diff
@@ -0,0 +1,172 @@
+# -*- coding: utf-8 -*-
+from dataiku.customrecipe import *
+import dataiku
+import pandas as pd
+from dataiku import pandasutils as pdu
+from dataiku.customrecipe import get_recipe_config
+from language_dict import SUPPORTED_LANGUAGES_SPACY
+from dku_config import DkuConfig
+
+# All plugin parameters name
+MATCHING_PARAMS = ["case_sensitivity", "lemmatization", "unicode_normalization"]
```

Comment:
Can you make this into a class constant? Just declare it inside the class. That way it's easier to know the scope of this constant

Reply:
here : 1016248